starport/pkg/cosmosgen/templates/pinia/index.ts.tpl

```diff
@@ -0,0 +1,26 @@
+// THIS FILE IS GENERATED AUTOMATICALLY. DO NOT MODIFY.
+
+import { defineStore } from 'pinia'
+
+import { 
+{{ range .Module.Types }}{{ .Name }},
+{{ end }}
+} from "ts-client/{{ .Module.Pkg.Name }}/types";
```

Comment:
```suggestion
} from "../../ts-client/{{ .Module.Pkg.Name }}/types";
```

Reply:
The generated packages, client & vue, are going to be `npm installed`